rpool.go

```diff
@@ -236,7 +231,6 @@ func (p *Pool) manage() {
 				r <- c.resource
 				resources = resources[:cl-1]
 				out++
-				stats.BumpSum(p.Stats, "acquire.pool", 1)
```

Comment:
can we keep this one, just in case we want to know what fleet is consuming a large number of connections?